shopify/mixins.py

```diff
@@ -11,8 +11,15 @@ def count(cls, _options=None, **kwargs):
 
 class Metafields(object):
 
-    def metafields(self):
-        return shopify.resources.Metafield.find(resource=self.__class__.plural, resource_id=self.id)
+    def metafields(self, _options=None, **kwargs):
+        if _options is None:
+            _options = kwargs
+        return shopify.resources.Metafield.find(resource=self.__class__.plural, resource_id=self.id, **_options)
+
+    def metafields_count(self, _options=None, **kwargs):
+        if _options is None:
+            _options = kwargs
```

Comment:
I took the pattern from the `count` method of the `Countable` class, seen just above in the same file. Basically, it allows you to either specify parameters with a dictionary or with keyword arguments. This is consistent with the behavior of the rest of the API.